README.md

```diff
@@ -7,916 +7,62 @@
-            user: <username>
-
-            # (Optional) User ID for starting command as a different user. If
-            # both user and user-id are specified, the user's UID must match
-            # user-id.
-            user-id: <uid>
-
-            # (Optional) Group name for starting command as a different user.
-            # It is an error if the group doesn't exist.
-            group: <group name>
-
-            # (Optional) Group ID for starting command as a different user. If
-            # both group and group-id are specified, the group's GID must
-            # match group-id.
-            group-id: <gid>
-
-            # (Optional) Working directory to run command in. By default, the
-            # command is run in the service manager's current directory.
-            working-dir: <directory>
-
-# (Optional) A list of remote log receivers, to which service logs can be sent.
-log-targets:
-
-  <log target name>:
-
-    # (Required) Control how this log target definition is combined with
-    # other pre-existing definitions with the same name in the Pebble plan.
-    #
-    # The value 'merge' will ensure that values in this layer specification
-    # are merged over existing definitions, whereas 'replace' will entirely
-    # override the existing target spec in the plan with the same name.
-    override: merge | replace
-
-    # (Required) The type of log target, which determines the format in
-    # which logs will be sent. The supported types are:
-    #
-    # - loki: Use the Grafana Loki protocol. A "pebble_service" label is
-    #   added automatically, with the name of the Pebble service as its value.
-    type: loki
-
-    # (Required) The URL of the remote log target.
-    # For Loki, this needs to be the fully-qualified URL of the push API,
-    # including the API endpoint, e.g.
-    #     http://<ip-address>:3100/loki/api/v1/push
-    location: <url>
-
-    # (Optional) A list of services whose logs will be sent to this target.
-    # Use the special keyword 'all' to match all services in the plan.
-    # When merging log targets, the 'services' lists are appended. Prefix a
-    # service name with a minus (e.g. '-svc1') to remove a previously added
-    # service. '-all' will remove all services.
-    services: [<service names>]
-
-    # (Optional) A list of key/value pairs defining labels which should be set
-    # on the outgoing logs. The label values may contain $ENV_VARS, which will
-    # be substituted using the environment for the corresponding service.
-    labels:
-      <label name>: <label value>
-```
-
-## API and clients
-
-The Pebble daemon exposes an API (HTTP over a unix socket) to allow remote clients to interact with the daemon. It can start and stop services, add configuration layers the plan, and so on.
+You can also follow the [Getting started with Pebble tutorial](https://canonical-pebble.readthedocs-hosted.com/en/latest/tutorial/getting-started/).
```

Comment:
What about "For a slightly longer introduction, you can also go through the [official tutorial](#)."